common/pb/serverinfo_user.proto

```diff
@@ -11,6 +12,10 @@ message ServerInfo_User {
         Male = 0;
         Female = 1;
     };
+    enum UserRole {
```

Comment:
The enum?  Its not even used for any of the code.  I origiinally used it and it did make it hard to extend the user roles instead of a basic int so I left it in for human reference for what role is what.